src/serac/physics/utilities/functional/tensor.hpp

```diff
@@ -148,7 +147,7 @@ struct tensor<T, first, rest...> {
    * @brief The array of dimensions containing the shape (not the data itself)
    * Similar to numpy.ndarray.shape
    */
-  static constexpr int shape[ndim] = {first, rest...};
+  static constexpr int first_dim = first;
```

Comment:
for context, this change was made because only `shape[0]` was being used, and nvcc was having trouble compiling `static constexpr` C-array members